WordPress/gutenberg
# Block bindings: Lock editing in fields in editor if meta fields panel is opened.

## What?

Fixes #64217
Following @gziolo [proposal](https://github.com/WordPress/gutenberg/issues/64217#issuecomment-2306716812), let's try an experiment that locks editing if the meta fields panel is opened.
 
An alternative to #64505

## Why?

Experimenting.

## How?


## Testing Instructions




1) Open a post.
2) Enable custom fields in preferences. Check that the note is added when custom fields are enabled.
3) Bind some connections, check that are not editable.


## Screenshots or screencast 

https://github.com/user-attachments/assets/b203c55f-eae3-46b4-b095-31699dadf0fc
